parcel.go

```diff
@@ -13,48 +13,74 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		return 0, err
+	}
+	i, err := res.LastInsertId()
+
+	return int(i), err
```

Comment:
Можно лучше: так делать можно и это не ошибка, но лучше все так делать отдельную проверку, так как функция может меняться и эта проверку забудут добать
```suggestion
        if err != nil {
            return 0, err
        }
        return int(id), nil
```